src/omero_zarr/masks.py

```diff
@@ -392,11 +452,11 @@ def masks_to_labels(
         fillColors: Dict[int, str] = {}
         for count, shapes in enumerate(masks):
```

Comment:
One more "masks" here :smile:

Reply:
Guess then the whole class could have been renamed to "shapes"...

Reply:
Or renamed to Labels, since that's what it generates?
I don't know if in future we'll have output that generates other formats - e.g. polygons in OME-Zarr?
Then the commands and classes might be named by what they output, rather than input?